doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -109,7 +109,7 @@ Amazon Sidewalk
 Bluetooth® LE
 -------------
 
-|no_changes_yet_note|
+* The :c:func:`hci_vs_sdc_llpm_mode_set` function returns an error if Low Latency Packet Mode (LLPM) is not supported or not enabled in the Bluetooth LE controller configuration (:kconfig:option:`CONFIG_BT_CTLR_SDC_LLPM`).
```

Comment:
Behavior was changed, I wanted to notify customers (discussed with @rugeGerritsen, we think that known issue may not be necessary because it does not seem like a severe issue)